runtime/src/constants.rs

```diff
@@ -286,9 +281,9 @@ pub mod staking {
 		// Note: the EPM in this runtime runs the election on-chain. The election bounds must be
 		// carefully set so that an election round fits in one block.
 		pub ElectionBoundsMultiPhase: ElectionBounds = ElectionBoundsBuilder::default()
-			.voters_count(10_000.into()).targets_count(1_500.into()).build();
+			.voters_count(MaxElectingVoters::get().into()).build();
 		pub ElectionBoundsOnChain: ElectionBounds = ElectionBoundsBuilder::default()
-			.voters_count(5_000.into()).targets_count(1_250.into()).build();
+			.voters_count(5_000.into()).build();
```

Comment:
Could we also define `5_000` as a constant too?

Reply:
yeah, will do that